scripts/west_commands/runners/jlink.py

```diff
@@ -151,7 +151,7 @@ def read_version(self):
     def supports_nogui(self):
         ver = self.read_version()
         # -nogui was introduced in J-Link Commander v6.80
-        return version.parse(ver) >= version.parse("6.80")
+        return version.LegacyVersion(ver) >= version.LegacyVersion("6.80")
```

Comment:
LegacyVersion is deprecated. I think we need to come up with another solution.

Reply:
> I think we need to come up with another solution.

Let's just do something J-Link specific then.

Reply:
I'll take a look at this sometime tomorrow.

Reply:
Strange because this works locally for me with 7.11b

Reply:
I have just tested with `SEGGER J-Link Commander V7.20 (Compiled Apr 28 2021 17:38:41)` without encountering any issues. It is correctly using `-nogui 1`.

Reply:
I've got a solution and will send a PR

Reply:
A fix which works on Windows is submitted as part of https://github.com/zephyrproject-rtos/zephyr/pull/34660. It relies on a third-party library to get the version from segger's shared library API. See ` runners: jlink: replace version parsing code ` in that PR.